tests/SandboxRequire.spec.ts

```diff
@@ -1,3 +1,4 @@
+import * as path from 'path';

```

Comment:
Nah, there is actually no "static" convention.

The actual convention here was, that imports, which include with an `* as` , should be at the beginning of the document. (This is also forced  by my IDE)

- `* as NAME from`
- `{} from `
- CommonJS

I don't think it isn't necessary to change all other files. We could just do this by time automatically. Not a high priority.